### PR TITLE
* Resolves #1249

### LINKED ISSUE #1249: [Bug]: Ribbon Form Bars are no longer Drawn with the theme colouring
**Describe the bug**
image

**To Reproduce**
It's been like this since 2023-05-21 (The last time I created a release build of the Ribbon Gallery Example)

- In the design (Using todays Alpha) you can see the Theme colouring,
- When the application is run, it has the OS's Control Title frame BUT with the Orb drawn over it !

### PR DESCRIPTION
**Pull request by Wagnerp**
* Resolves #1249 

image

Needs to be tested

### CODE REVIEW

File: Source/Krypton Components/Krypton.Ribbon/Controller/GalleryButtonController.cs [hunk 1]
```diff
@@ -25,7 +25,8 @@ internal class GalleryButtonController : GlobalId,
         private bool _pressed;
         private bool _mouseOver;
         private NeedPaintHandler? _needPaint;
-        private readonly Timer _repeatTimer;
+        private readonly Timer? _repeatTimer;
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Ribbon/Controller/GalleryButtonController.cs:28]: Why is this nullable ?
If it is set to null in Dispose, then that's okay !

Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Ribbon/Controller/GalleryButtonController.cs:28]: It crashed in testing. Maybe use `!`?

Comment (Smurf-IV) [reply Source/Krypton Components/Krypton.Ribbon/Controller/GalleryButtonController.cs:28]: That means something else is wrong.

Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Ribbon/Controller/GalleryButtonController.cs]: Now fixed, the timer wasn't disposed of

**Pull request merged**